File: libgomp/testsuite/libgomp.oacc-fortran/data-already-1.f
! { dg-skip-if "" { *-*-* } { "*" } { "-DACC_MEM_SHARED=0" } }

      IMPLICIT NONE
      INCLUDE "openacc_lib.h"

      INTEGER I

      CALL ACC_COPYIN (I)
      WRITE(0, *) "CheCKpOInT"
!$ACC DATA COPY (I)
      I = 0
!$ACC END DATA

      END

! { dg-output "CheCKpOInT(\n|\r\n|\r).*" }

File: libgomp/testsuite/libgomp.oacc-c-c++-common/data-already-1.c
/* { dg-skip-if "" { *-*-* } { "*" } { "-DACC_MEM_SHARED=0" } } */

#include <stdio.h>
#include <openacc.h>

int
main (int argc, char *argv[])
{
  int i;

  acc_copyin (&i, sizeof i);

  fprintf (stderr, "CheCKpOInT\n");
#pragma acc data copy (i)
  ++i;

  return 0;
}

/* { dg-output "CheCKpOInT(\n|\r\n|\r).*" } */
